aws/src/main/java/org/apache/iceberg/aws/s3/S3OutputFile.java

```diff
@@ -43,6 +44,11 @@ public static S3OutputFile fromLocation(String location, S3Client client, AwsPro
     return new S3OutputFile(client, new S3URI(location), awsProperties, metrics, writeTags);
   }
 
+  public static S3OutputFile fromLocation(String location, S3Client client, AwsProperties awsProperties,
```

Comment:
the `fromLocation` at L42 was just added, not released yet, we can just update that method to also include this new field instead of creating a new static helper.